collect_stata/write_json.py

```diff
@@ -202,10 +202,44 @@ def generate_statistics(
 
     return metadata
 
+def update_metadata(
+    metadata: List[Dict[str, Union[str, Dict[str, List[Union[int, str, bool]]]]]],
+    metadata_de: List[Dict[str, Union[str, Dict[str, List[Union[int, str, bool]]]]]],
+) -> List[Dict[str, Union[str, Dict[str, List[Union[int, str, bool]]]]]]:
+    """Gather information of german and english metadata and create a new metadata variable
+
+    Input:
+    metadata: Metadata of the english imported data.
+    metadata_de: Metadata of the german imported data.
+
+    Output:
+    metadata: Metadata variable with german and english labels if given.
```

Comment:
Please use Args: and Returns. Please indent blocks after `:`
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html